src/test/resources/edu/cornell/kfs/module/purap/jaggaer/xml/cu-spring-jaggaer-test.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+        http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+    <bean id="dateTimeService" class="edu.cornell.kfs.sys.service.impl.TestDateTimeServiceImpl"/>
+
+    <bean id="jaggaerXMLInputFileType"
```

Comment:
To simplify the bean setup for these tests and to minimize bean duplication, I would suggest structuring the file as follows:

1. Add `<import>` elements for each non-unit-test Spring file that contains beans needed for your test.
2. Add this tag:
   ```xml
   <import resource="classpath:edu/cornell/kfs/sys/cu-spring-base-test-beans.xml"/>
   ```
3. (Optional) If you want to automatically set up a few common test overrides of base beans (including a test "dateTimeService" override), then add this tag:
   ```xml
   <import resource="classpath:edu/cornell/kfs/sys/cu-spring-sys-test.xml"/>
   ```
4. If you have other test-only Spring XML files that you want to import, add their imports after the ones from above.
5. If the needed beans contain property substitution placeholders (in `${propname}` format), add an override of the "propertyPlaceholderConfigurer" bean and merge in the property key-value pairs that you want to use. (See `cu-spring-rass-service-test.xml` for an example.)
6. Create new beans or override imported beans as needed.
7. To limit which beans will actually be loaded by the unit test, add an override of the "beanFilterPostProcessor" bean and merge in a set of bean names that you want the test to preserve. Make sure you specify `merge="true"` on the overridden set, since the parent bean already contains some entries for preserving the base test beans. (See `cu-spring-rass-service-test.xml` for an example.)

Reply:
All I need for the test is dateTimeService and jaggaerXMLInputFileType. Can I import the cu-spring-purap.xml file from the non-test resources directory?

Reply:
Yes, you can import `cu-spring-purap.xml`, just make sure the import precedes those for the test XML files.